index.html

```diff
@@ -9,6 +9,16 @@
 </head>
 <body>
     <div class="home-bg">
+      <div class="mobile-menu">
+        <div class="close-button">
+          <img src="./assets/close-button.png" alt="">
+        </div>
+        <ul class="mobile-menu-items">
+          <li class="mobile-menu-item"><a href="#portfolio" class="mobile-menu-link">Portfolio</a></li>
+          <li class="mobile-menu-item"><a href="#about" class="mobile-menu-link">About</a></li>
+          <li class="mobile-menu-item"><a href="#contact" class="mobile-menu-link">Contact</a></li>
+        </ul>
```

Comment:
Good job so far 👍🏽 but the rest of the page is showing while the menu is open, you can either make it take the full height of the screen or just make the rest of the page unscrollable while it's opened 😉 take a look at the design issue it gives 👇🏽 

![Screenshot 2023-05-16 at 07 31 34](https://github.com/Peter-Kibet/Peter-K-Portfolio/assets/45739963/b78ab915-8265-485a-b755-1d910dbf32f8)